package.json

```diff
@@ -25,6 +26,9 @@
   },
   "homepage": "https://github.com/sprjr/react-access#readme",
   "files": ["dist"],
+  "peerDependencies": {
+    "react": "^15.3.2"
```

Comment:
was wondering about that, but wanted to actually asses what the full spectrum of compatibility was - and figured I would tackle that in Travis and tests as well